README.md

```diff
@@ -109,35 +153,131 @@ Now in a slurm job array the work will be performed (you can set the number of a
+cd ..
+```
+
+The sandbox directory now holds everything we need to send to the Grid worker nodes.
 
+Create the Tokens
+
+This example includes a bash script (``./createTokens``) that generates a sensible parameter file, with each line representing a set of parameters that the fractals program can be called with. Without arguments it creates a fairly sensible set of 24 lines of parameters. You can generate different sets of parameters by calling the program with a combination of ``-q``, ``-d`` and ``-m`` arguments, but at the moment no documentation exists on these. We recommend not to use them for the moment.
+
+* After you ran the ``createTokens`` script you'll see output similar to the following:
 ```
-wget https://files.pythonhosted.org/packages/7c/c8/f94a107eca0c178e5d74c705dad1a5205c0f580840bd1b155cd8a258cb7c/CouchDB-1.2.tar.gz
+./createTokens
+/tmp/tmp.fZ33Kd8wXK
+cat /tmp/tmp.fZ33Kd8wXK
 ```
+Now we will start using PiCaS. For this we need the downloaded CouchDB and PiCaS packages for Python and set the hostname, database name and our credentials for the CouchDB server:
 
-Now you can start the example from a grid login node with (in this case DIRAC is used for job submission):
+* Edit ``sandbox/picasconfig.py`` and set the PiCaS host URL, database name, username and password.
 
 ```
-dirac-wms-job-submit grid-example.jdl
+ln -s sandbox/picasconfig.py
 ```
 
-And the status and output can be retrieved with DIRAC commands, while in the token you see the status of the token and the tokens' attachments contain the log files. Once all tokens have been processed (this can be seen in the CouchDB instance) the grid job will finish.
+* Make the CouchDB package locally available:
+```
+tar -xvf sandbox/couchdb.tgz
+```
+
+* Upload the tokens:
+
+```
+$python createTokens.py /tmp/tmp.fZ33Kd8wXK
+```
+
+* Check your database in this link:
+```
+https://picas.surfsara.nl:6984/_utils/#/database/homerdb/_all_docs (replace homerdb with your Picas database name)
+```
+
+* Create the Views (pools) - independent to the tokens (should be created only once):
+
+```
+python createViews.py
+```
+To make use of the dirac tool, first source the dirac env
+
+```
+source /etc/diracosrc
+```
+
+* Create a proxy:
+```
+dirac-proxy-init -b 2048 -g lsgrid_user -M lsgrid --valid 168:00 # replace lsgrid with your VO
+```
+
+* Submit the pilot jobs:
+```
+dirac-wms-job-submit fractals.jdl -f jobIDs
+```
+
+
+It will recursively generate an image based on parameters received from PiCas. At this point, some of your tokens are processed on the Grid worker nodes and some of the tokens are already processed on the :abbr:`UI (User Interface)`. Note that the :abbr:`UI (User Interface)` is not meant for production runs, but only for testing few runs before submitting the pilot jobs to the Grid.
+
+* Convert the :abbr:`UI (User Interface)` output file to .png format and display the picture:
+```
+convert output_token_6 output_token_6.png # replace with your output filename
```

Comment:
this applies to all 3 scenarios of running the fractals example: local, slurm, grid